diffprivlib/tools/histograms.py

```diff
@@ -241,11 +258,14 @@ def histogramdd(sample, epsilon=1.0, bins=10, range=None, normed=None, weights=N
             dp_hist = dp_hist / np.diff(bin_edges[i]).reshape(shape)
         dp_hist /= dp_hist_sum
 
+    if accountant:
+        accountant.spend(epsilon, 0)
+
```

Comment:
I would change this code in the check_spend method of BudgetAccountant
This way we can have also the "no accountant" scenario

Reply:
Same as above, accountant now defaults to instance of `BudgetAccountant()`, so implicitly assuming accountant is of correct class, and no need for `if()`. b6ef353